src/rockstar-parser.peg

```diff
@@ -19,7 +19,7 @@ Pronoun = (
 
 TypeLiteral = v:TypeLiteralValue { return {t: 'Literal', v} }
 TypeLiteralValue =
-	('nothing'/'nobody'/'nowhere'/'empty'/'gone') {return 0} // TODO null?
+	('null'/'nothing'/'nobody'/'nowhere'/'empty'/'gone') {return 0}
```

Comment:
I don't think we can just set it to 0, the spec specifies elsewhere various things that would require keeping it as a separate value. Example:

> String \<plus\> Null =\> Convert the null to `null`